drop-in/src/main/java/com/adyen/checkout/dropin/internal/ui/DropInParamsMapper.kt

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2025 Adyen N.V.
+ *
+ * This file is open source and available under the MIT license. See the LICENSE file for more info.
+ *
+ * Created by oscars on 21/11/2025.
+ */
+
+package com.adyen.checkout.dropin.internal.ui
+
+import androidx.appcompat.app.AppCompatDelegate
+import com.adyen.checkout.core.components.CheckoutConfiguration
+import com.adyen.checkout.core.sessions.CheckoutSession
+import java.util.Locale
+
+internal class DropInParamsMapper {
+
+    fun map(
+        checkoutConfiguration: CheckoutConfiguration,
+        checkoutSession: CheckoutSession?,
```

Comment:
Is there a reason you prefer not to use `SessionParams` here?

Reply:
I wasn't aware of `SessionParams`. I'll update to use that instead